README.md

```diff
@@ -1,24 +1,146 @@
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ERB</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- rails react backend api 
+- Rails React Frontend App PR link: https://github.com/Profsain/hello-react-frontend/pull/1
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+>## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a computer and internet connection
+- Have `Ruby` installed on your computer
+- Have a basic knowledge of `Ruby` and `OOP` concept
+- Have a general understanding of what testing is
+- Have `visual-studio code` or any other code editor installed on your computer.
+
+### Setup
+- In order to get a copy of this project you need to download it from https://github.com/Profsain/hello-rails-react-backend
+- Extract the zipped file and open it in your code editor
+### Install
+- Run the command bellow in your terminal to get all required files 
+``` Ruby
+bundle install
+```
+
+### Run Server
+- Run the command bellow in your terminal
+  rails server
+```Ruby
+rails server
+```
+### Run tests
+you can run one of the following command in your terminal
+1. Run testing
+```Ruby
+rspec spec
+```
+2. Run linters
+```Ruby
+> Rubocop --color
+> Rubocop -A
+```
+## Authors
+
+👤 **Husseini Mudi Profsain**
+
+- GitHub: [@githubhandle](https://github.com/Profsain)
+- Twitter: [@twitterhandle](https://twitter.com/profsain)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/profsain)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Profsain/hello-rails-react-backend/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License
+
+This project is [MIT]() licensed.
```

Comment:
- [ ] Kindly provide the link to the MIT license to make your README look more professional.